src/LndClient/RPC/Katip.hs

```diff
@@ -227,14 +227,14 @@ catchWalletLock ::
 catchWalletLock env x = do
   x0 <- x
   case x0 of
-    Left LndWalletLocked -> do
-      $(logTM) (newSev env WarningS) "Wallet is locked, needs to be unlocked"
+    Left err -> do
```

Comment:
There is no specific error text, also use specific text is not relaible, so let's try to intercept any error and if it helps, okay, if not - function will return error.